README.md

```diff
@@ -113,6 +113,25 @@ App::make('Foo')->bar();
 
 For more information on what features are available on the `Swis\Laravel\Mautic\Client` class, check out the Mautic docs at [https://developer.mautic.org/#endpoints](https://developer.mautic.org/#endpoints), and the manager class at [https://github.com/GrahamCampbell/Laravel-Manager#usage](https://github.com/GrahamCampbell/Laravel-Manager#usage).
 
+# Notifications
+
+To use the ```NotificationDriver``` built into this package make sure the entity you want to notify has the following traits:
+```php
+class ClassName
+{
+    use Notifiable;
+    use SynchronizesWithMauticTrait;
+    use NotifiableViaMauticTrait;
+}
+```
+Then make sure to add a Notification to your Laravel project. This notification should include the ```MauticChannel``` from this package in the ```via()``` method. Make sure your notification includes a ```toMautic()``` method which returns an instance of ```MauticMessage```. For this you can use the ```create()``` method:
+```php
+public function toMautic(): MauticMessage
+    {
+        return MauticMessage::create($mauticId);
+    }
```

Comment:
I'd like to see a full notification class here, including the via method, etc. Also including how you can set the tokens and to.